tasks/89_symmertic-difference/symmertic-difference.js

```diff
@@ -0,0 +1,10 @@
+export const sym = (...args) => {
+  return args.reduce((cur, prev) => {
+    let arrayEl = [...new Set(prev), ...new Set(cur)]
```

Comment:
for this tasks you can work with `Sets` and sort only in the end when you need to return value, `Sets` has a few awesome methods for example `has`, `add` and so on